app/platform/templates/platform/developers/manage_section.html

```diff
@@ -0,0 +1,95 @@
+{% from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList %}
+{% from "govuk_frontend_jinja/components/button/macro.html" import govukButton %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "common/macros/move-up-down-table.html" import moveUpDownTable %}
+
+{% extends "platform/manage_grant_base.html" %}
+
+{% block pageTitle %}
+    {{ section.title }} - {{ collection.name }} - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% set active_sub_navigation_tab = "grant_developers" %}
+{% block beforeContent %}
+    {{ govukBackLink({
+        "text": "Back",
+        "href": url_for("platform.manage_collection", grant_id = grant.id, collection_id=collection.id)
+    }) }}
+{% endblock beforeContent %}
+
+{% block content %}
+<h1 class="govuk-heading-l">
+    <span class="govuk-caption-l">{{ collection.name }}</span>
+    {{ section.title }}
+</h1>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds govuk-!-margin-bottom-5">
+
+        {{ govukSummaryList({
+"rows": [{
+"key": {"text": "Title"},
+"value":{"text": section.title},
+"actions": {
+"items": [
+{
+"href": url_for("platform.edit_section", grant_id = grant.id, collection_id = collection.id, section_id = section.id),
+"text": "Change",
+"classes": "govuk-link--no-visited-state"
+}
+]
+}
+}]
+}) }}</div>
+</div>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <h2 class="govuk-heading-m">Forms</h2>
+        {% set forms_text %}
+        {% trans count=section.forms | length %}
+            {{ count }} form
+            {% pluralize %}
+            {{ count }} forms
+        {% endtrans %}
+        {% endset %}
```

Comment:
The multiline of this set is adding some extra spacing that it'd be nice to strip

```suggestion
        {% set forms_text %}{% trans count=section.forms | length %}{{ count }} form{% pluralize %}{{ count }} forms{% endtrans %}{% endset %}
```

Is the {% trans %} tag just to support pluralizing rather than actual translations?